apps/frontend/src/libs/components/header/styles.module.css

```
@@ -0,0 +1,28 @@
+.header {
+	box-sizing: border-box;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	width: 100%;
+	padding: 12px 32px 12px 48px;
+	border-bottom: 1px solid var(--color-stroke-weak);
+}
+
+.userInfo {
```

Let's use kebab-case for CSS class names. The same is for avatar image or other class names 
```suggestion
.user-info {
```